testData/dbData/views.json

```diff
@@ -10,6 +10,15 @@
             },
             "findAuthorizationByAccessToken": {
                 "map": "function(doc) {if (doc.type === 'gpiiAppInstallationAuthorization' && doc.revoked === false) {emit(doc.accessToken, {'_id': doc.clientId, 'authorization': doc})}}"
+            },
+            "listKeysForPrefsSafe": {
```

Comment:
Most view names starts with "find" and these 2 use "list". Is there a reason in using a different word? If not, can you sync up the view names?

Reply:
I think this makes sense @cindyli . I will probably leave `listPrefsSafes` as it is, since there is not search argument given to it, it really is just a listing.